cmd/sonobuoy/app/pluginCache.go

```diff
@@ -38,20 +39,27 @@ var (
 	defaultSonobuoyDir = filepath.Join("~", ".sonobuoy")
 )
 
+// pluginNotFoundError is a custom type so we can tell whether or not loading the plugin
+// from the installation directory FAILED or if it just wasn't found.
+type pluginNotFoundError struct{ e error }
+
+func (p *pluginNotFoundError) Error() string {
+	return p.e.Error()
+}
+
 func NewCmdPlugin() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "plugin",
 		Aliases: []string{"plugins"},
 		Short:   "Manage your installed plugins",
-		Hidden:  true,
 	}
 
 	listCmd := &cobra.Command{
 		Use:   "list",
 		Short: "List all installed plugins",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return listInstalledPlugins(getPluginCacheLocation(cmd))
```

Comment:
Thought I'd have a flag potentially to impact cache usage but opted out of that. Removed unnecessary arg.